suricata/update/commands/listsources.py

```diff
@@ -72,11 +71,10 @@ def list_sources():
 
     free_only = config.args().free
     if not sources.source_index_exists(config):
-        logger.info("No source index found, running update-sources")
-        try:
-            update_sources()
-        except exceptions.ApplicationError as err:
-            logger.warning("%s: will use bundled index.", err)
+        logger.warning("Source index does not exist, will use bundled one.")
+        logger.warning("Please run suricata-update update-sources.")
+        return 1
```

Comment:
We don't need to return here. The rest of the code will run fine with the "bundled" index. It may be out of date, but we thats what this warning is for.

Reply:
Thanks! I removed the return statement, but the checks are still failing.

Reply:
> Thanks! I removed the return statement, but the checks are still failing.

Just looking at this now.  I think we have an integration test that checks for the index.yaml being downloaded after a list-sources, this will have to be changed - will confirm.

Reply:
Ok, to tests/integration_tests.py checks for the index.yaml at line 66, to verify that it was downloaded as part of a list source.  With this change, we should verify that it was *not* downloaded.

Reply:
Thanks for the information. Sorry for the multiple pull requests. Is there any chance I could run the checks locally before opening a pull request?